src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -42,27 +40,27 @@ class Ledger(vm: VM) extends Logger {
     * This function runs transaction
     *
     * @param block
+    * @param blockchain
     * @param storages
-    * @param stateStorage
+    * @param validators
     */
   private def executeBlockTransactions(
     block: Block,
     blockchain: Blockchain,
     storages: BlockchainStorages,
-    stateStorage: NodeStorage):
+    validators: Validators):
```

Comment:
That makes a lot of sense :+1: I did this as it was previously done with the `blockValidator` function (in `SyncController`) only for simpler testing, but it seems it would also work for the config you are working in :smile: